pages/reference.js

```diff
@@ -41,6 +44,11 @@ export default function WordsPage() {
             {brailleCharJsx(brailleCode(char))}
           </Box>
         </TableCell>
+        <TableCell>
+          <Box className={semaphore_styles.result_input_char}>
```

Comment:
Přijde mi názornější, když jsou vidět nejen ruce, ale i čtverec vytyčující plátno semaforu, jako je to na stránce se semaforem.